docs/Operators.md

```diff
@@ -2371,6 +2371,72 @@ Other versions of this operator: <a href="Changelog.md#GRU-1">GRU-1</a>
 </dl>
 
 
+#### Examples
+
+<details>
+<summary>defaults</summary>
+
+```python
+input = np.array([[[1., 2.], [3., 4.], [5., 6.]]]).astype(np.float32)
+
+input_size = 2
+hidden_size = 4
+
+node = onnx.helper.make_node(
+    'GRU',
+    inputs=['X', 'W', 'R'],
+    outputs=['Y'],
+    hidden_size=hidden_size
+)
+
+W = np.ones((1, 3 * hidden_size, input_size)).astype(np.float32)
+R = np.ones((1, 3 * hidden_size, hidden_size)).astype(np.float32)
+
+output = np.array([[0.04719133, 0.04719133, 0.04719133, 0.04719133],
+                   [0.04791026, 0.04791026, 0.04791026, 0.04791026],
+                   [0.04792343, 0.04792343, 0.04792343, 0.04792343]]).astype(np.float32)
+
+expect(node, inputs=[input, W, R], outputs=[output], name='test_gru_defaults')
+```
+
+</details>
+
+
+<details>
+<summary>initial_bias</summary>
+
+```python
+input = np.array([[[1., 2.], [3., 4.], [5., 6.]]]).astype(np.float32)
+
+input_size = 2
+hidden_size = 4
+custom_bias = 0.1
+
+node = onnx.helper.make_node(
+    'GRU',
+    inputs=['X', 'W', 'R', 'B'],
```

Comment:
Are we planning to add more coverage for different activation functions?

Reply:
The scope of this PR is to provide basic test coverage for forward case, as well as reference numpy implmentation for it. Different variations shall be added seperately as part of other PRs.

Reply:
OK.